src/core/ref.jl

```diff
@@ -136,3 +136,14 @@ function _calc_tp_trans_Tvi(pm::_PMs.GenericPowerModel, i::Int; nw=pm.cnw)
     Cv_to *= vmult
     return (Tv_fr,Tv_im,Ti_fr,Ti_im,Cv_to)
 end
+
+
+function make_multiconductor!(mp_data, n_conductors::Int)
```

Comment:
I think this function should go into `src/core/data.jl`

Reply:
correct!

Reply:
good catch